### ISSUE TITLE
Print to pdf fails to generate file using system installed font

### ISSUE THREAD

**Issue by letroisieme** [ONLYOFFICE/DesktopEditors#1172]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

ONLYOFFICE Desktop Editors version 7.2.1.36 (x64 exe package)

### Expected Behavior

I am using ONLYOFFICE desktop editor linked to an ONLYOFFICE personal workspace.

I installed a custom font (bulmer BT) to my system and am able to use it in the desktop editor, but not on the browser workspace;

I wanted to export a pdf version of my document using "print to pdf" and expected it to reflect what I was working on in onlyoffice desktop editor

### Actual Behavior

The exported pdf is broken and uses a random font, the azerty accents (à é è) are broken too

### Reproduction Steps

1 Install onlyoffice desktop editor
2 link it to a onlyoffice personal workspace
3 Install a custom font and use it in desktop editor
4 try to export to pdf

### Additional information

I suspect the app is just showing me the file using the font while it is not actually applied to the original file on the cloud and there is a conflict while generating the pdf

How can I get it to generate the file accordingly? Thanks for any help

**Comment by Rita-Bubnova** [comment]
Hello, @letroisieme.
Thank you for the report. 

I think this problem can be fixed as a solution at https://github.com/ONLYOFFICE/DocumentServer/issues/1244. 
I add your problem in issue 50033 in our private issue tracker (more details in [the comment](https://github.com/ONLYOFFICE/DocumentServer/issues/1244#issuecomment-827455068)).

**Comment by letroisieme** [comment]
Hello, @Rita-Bubnova 
Thanks for your answer.

I did read the thread you linked before posting and if I understand it correctly that means there is no way to export the file I'm looking at in desktop editors as a pdf? I have to use a default font or migrate the whole thing to another text editor and export it here?

If I can suggest something, maybe either make it possible to generate the file from the current state of the file locally, so that we can export something that is similar to what we're working on, or disable altogether the possibility to use custom system installed font; 
As it stands, the app is misleading and counterintuitive by giving you the possibility to work with such font but not export the finished file and results in situations like the one I'm in :  a new user tries to work with the app only to face that issue at the last minute

**Comment by Rita-Bubnova** [comment]
> For now as a workaround I found that I can download the file as docx, open it again in desktop editors and from there print as a pdf

You're right. At the moment you can use ONLYOFFICE Desktop Editors as a workaround.

**Comment by letroisieme** [comment]
Thing is, if I'm not mistaken, you cannot export a file that is stored on the cloud but being worked on in the desktop editor, one needs to download the docx file, open it again as a local file and then save it as a pdf for it to work; not too intuitive but it ended up working
Thanks for the support have a nice dau
